src/flexalloc_mm.c

```diff
@@ -1377,7 +1377,7 @@ fla_object_read(const struct flexalloc * fs, struct fla_pool const * pool_handle
   int err;
   uint64_t obj_eoffset, obj_soffset, r_soffset, r_eoffset, slab_eoffset;
   struct fla_pool_entry *pool_entry = &fs->pools.entries[pool_handle->ndx];
-  struct fla_sync_strp_params sp;
+  struct fla_strp_params sp;
```

Comment:
I am stuck in C89 due to previous kernel development. They are making the switch and we don't need to stay on C89 lets move this struct into the scope of call of fla_xne_async_strp_seq_x

Reply:
Seems reasonable as we do not use that variable in any other scope within that function